CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+## v1.5.0 / 2018-01-10
```

Comment:
This is actually 2019-01-10. :)

Reply:
oops, good catch! :slightly_smiling_face: